ophirofox/content_scripts/lobs.js

```diff
@@ -0,0 +1,15 @@
+const title = document.querySelector(".article-page-header");
+    
+function findPremiumBanner() {
+    if (!title) return null;
```

Comment:
pas besoin d'être global 

```suggestion
function findPremiumBanner() {
    const title = document.querySelector(".article-page-header");
    if (!title) return null;
```